### PR TITLE
Send mail to the support

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: gui/lib/Help.elm [hunk 1]
```diff
@@ -124,6 +124,33 @@ view helpers model =
                     ]
                 ]
             , h2 [] [ text (helpers.t "Help Official Support") ]
+            , if model.status == Success then
+                p [ class "message--success" ]
+                    [ text (helpers.t "Help Your mail has been sent. We will try to respond to it really soon!") ]
```
Comment (sebn) [inline gui/lib/Help.elm]: Will the `model.status` be reset at some point, allowing me to send another email?
Or should we add some `Send another mail` button?

File: gui/lib/Help.elm [hunk 2]
```diff
@@ -124,6 +124,33 @@ view helpers model =
...
+              else
+                Html.form [ class "send-mail-to-support" ]
+                    [ case model.status of
+                        Error error ->
+                            p [ class "message--error" ]
+                                [ text ("Error: " ++ error) ]
```
Comment (sebn) [inline gui/lib/Help.elm]: Will this be a low-level error message? Should we show it to the user, or log it and display a more user-friendly message?

File: gui/lib/Help.elm (hunk 1 above)
Comment (nono) [inline gui/lib/Help.elm]: You can close and reopen the window ;-)
I think we can improve this after the redesign of the GUI.

Comment (sebn) [inline gui/lib/Help.elm]: That's what I was thinking of 👍 
It's so nice to get this feature back!

**Pull request merged**